core/src/main/scala/org/apache/spark/api/r/RBackend.scala

```diff
@@ -47,10 +48,8 @@ private[spark] class RBackend {
 
   def init(): (Int, RAuthHelper) = {
     val conf = new SparkConf()
-    val backendConnectionTimeout = conf.getInt(
-      "spark.r.backendConnectionTimeout", SparkRDefaults.DEFAULT_CONNECTION_TIMEOUT)
```

Comment:
the whole file `SparkRDefaults.scala` looks not being referred anymore. Can we delete this file?

Reply:
Nice finding! Will remove.